settings_snowDA

```diff
@@ -4,8 +4,8 @@
 # if calling from submit_cycle,vars in this section will already be set. Otherwise, need to be set here. 
```

Comment:
should be 'main.sh' not submit_cycle? or did you mean submit_job?

Reply:
Changed to "if calling from land-offline_workflow". This comment is included in the settings template file that is included in the land-offline_workflow.